content/guides/astro.md

```diff
@@ -17,16 +17,164 @@ comments: false
 draft: false
 ---
 
+{{< hextra/hero-subtitle >}}
+  Astro is an all-in-one web framework for building content-driven websites, that pioneers a new frontend architecture to reduce JavaScript overhead and complexity. Learn in this guide how to deploy an Astro site on Clever Cloud.
+{{< /hextra/hero-subtitle >}}
+
+## Overview
+
+Clever Cloud supports deploying both fully static and on-demand rendered Astro projects. Regardless of your `output` mode ([pre-rendered or on-demand](/en/basics/rendering-modes/)), you can choose to deploy as a **static application** which runs using a post-build hook, or as a **Node.js** application, which works out-of-the-box with your `package.json`.
```

Comment:
As suggested by @sardemff7 in t[his commit](https://github.com/CleverCloud/documentation/commit/acd57a05b8a28934d4a94add7bbd2e6255e2412b#diff-f45ef70a1d05b0377e552d9bc23317bfdf8cfcf2f12bea7e41bfd46214ce27a5R37), because it summarizes nicely the purpose of each configuration.

Really love the package automation idea in your branch but I see some difficulty to implement it on guides, because often the project configuration is too specific to enable templating.

However, I think it's an idea we should explore.

```suggestion
Clever Cloud supports deploying both [fully static and on-demand rendered](https://docs.astro.build/en/basics/rendering-modes/) Astro projects:
- The `static` output mode is ideal for most content-oriented website, for which you have no need for per-visitor server-side customisation. Consider using a [Static runtime](/doc/applications/static/) when using this output mode, with the site generation in a post-build hook.
-  The `server` or `hybrid` output modes: consider using a [Node.js runtime](/doc/applications/javascript/nodejs/) runtime with [Astro’s Node adapter](https://docs.astro.build/en/guides/integrations-guide/node/)
```